### ISSUE TITLE
Add smsdbuser and smsdbrole descriptions

### ISSUE THREAD

**Issue by brmcmill** [MicrosoftDocs/SCCMdocs#1160]
I think we should add the following User and Database Roles with a Description of each within SQL to this article:
https://docs.microsoft.com/en-us/sccm/core/plan-design/hierarchy/accounts

I have had experiences where customers are hardening their SQL environments and have questioned what these account/roles are used for within ConfigMgr.

**Users:**
smsdbuser_ReadOnly
smsdbuser_ReadWrite
smsdbuser_ReportSchema

**Database Roles:**
smsdbrole_AITool
smsdbrole_AIUS 
smsdbrole_AMTSP
smsdbrole_CRP
smsdbrole_CRPPfx
smsdbrole_DMP
smsdbrole_DmpConnector
smsdbrole_DViewAccess
smsdbrole_DWSS
smsdbrole_EnrollSvr
smsdbrole_extract
smsdbrole_MCS
smsdbrole_MP
smsdbrole_MPUserSvc
smsdbrole_siteprovider
smsdbrole_siteserver
smsdbrole_SUP
smsdbrole_WebPortal
smsschm_users

I would be more than happy to update the Doc or another article that is more appropriate if it's possible to have these publicly detailed.

**Comment by aczechowski** [comment]
Thanks @brmcmill, great idea. Go ahead and submit an edit if you want. For more information, see [How to contribute](https://docs.microsoft.com/sccm/core/understand/use-docs#bkmk_contribute).

**Comment by brmcmill** [comment]
Thanks @aczechowski, I'll see if I get some help on the explanations of each.

**Comment by brmcmill** [comment]
Hey @aczechowski, apologies for the delay.  I made a doc edit.  I plan to add the SQL database roles later if this is suitable for this page.  Hope this helps.  Here's an outline of the changes:

Added the following section:
- User Objects that Configuration Manager uses in SQL 

Modified the text at the top of the page so it outlines the inclusion of SQL objects:
- Description: "Identify and manage the Windows groups, accounts, and SQL objects used in Configuration Manager."
- Applies To: "Use the following information to identify the Windows groups, accounts, and SQL objects that are used in Configuration Manager, how they are used, and any requirements. "

Next plan: Add SQL Database Roles and their Description.  I first wanted to ensure this would meet the criteria as requested.
